planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -93,6 +93,18 @@ bool AvoidanceModule::isExecutionRequested() const
     return true;
   }
 
+  // Check ego is in preferred lane
+  const auto current_lanes = util::getCurrentLanes(planner_data_);
+  lanelet::ConstLanelet current_lane;
+  lanelet::utils::query::getClosestLanelet(
+    current_lanes, planner_data_->self_pose->pose, &current_lane);
```

Comment:
```planner_data_->self_pose->pose```

In the latest code, this has been replaced with ```planner_data_->self_odometry->pose.pose```

Reply:
Done
https://github.com/autowarefoundation/autoware.universe/pull/2781/commits/8f9311f6e4f3f4dbff12fa679c7d32f033b272a4